src/CONST.js

```diff
@@ -2542,6 +2546,11 @@ const CONST = {
     TRANSLATION_KEYS: {
         ATTACHMENT: 'common.attachment',
     },
+    TAB: {
+        RECEIPT_TAB_ID: 'ReceiptTab',
+        TAB_MANUAL: 'manual',
+        TAB_SCAN: 'scan',
```

Comment:
NAB: Let's jus call these MANUAL/SCAN, since they'll be TAB.TAB_MANUAL if we keep them like this